std-bits/table/src/main/java/org/enso/table/write/DelimitedWriter.java

```diff
@@ -0,0 +1,191 @@
+package org.enso.table.write;
+
+import org.enso.table.data.table.Table;
+import org.enso.table.formatting.DataFormatter;
+
+import java.io.IOException;
+import java.io.Writer;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZonedDateTime;
+
+public class DelimitedWriter {
+  private static final char NEWLINE = '\n';
```

Comment:
just to minimise the work later - this might as well be a member field we pass from Enso verse.